### PR TITLE
Add DevOps Course

### PR DESCRIPTION
**Pull request by DrishtiJ**
## What does this PR do?
Add resource

### Description
DevOps Course

### Why is this valuable (or not)?
Learn Devops

### How do we know it's really free?
Listed as free course on Udacity 

### For book lists, is it a book? For course lists, is it a course? etc.
Course

## Checklist:
- [x] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md).
- [x] [Search](https://ebookfoundation.github.io/free-programming-books-search/) for duplicates.
- [x] Include author(s) and platform where appropriate.
- [x] Put lists in alphabetical order, correct spacing.
- [x] Add needed indications (PDF, access notes, under construction).
- [x] Used an informative name for this pull request.

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

File: courses/free-courses-en.md [hunk 1]
```diff
@@ -372,6 +372,7 @@
 ### DevOps
 
 * [DevOps Bootcamp](https://www.youtube.com/playlist?list=PL9gnSGHSqcnoqBXdMwUTRod4Gi3eac2Ak) - Kunal Kushwaha
+* [Intro to DevOps](https://www.udacity.com/course/intro-to-devops--ud611) - Udacity
```
Comment (eshellman) [inline courses/free-courses-en.md:375]:
```suggestion
* [Intro to DevOps](https://www.udacity.com/course/intro-to-devops--ud611) - Nutani (Udacity)
```

Comment (DrishtiJ) [reply courses/free-courses-en.md]: Done.

File: courses/free-courses-en.md [hunk 2]
```diff
@@ -372,6 +372,7 @@
 ### DevOps
 
 * [DevOps Bootcamp](https://www.youtube.com/playlist?list=PL9gnSGHSqcnoqBXdMwUTRod4Gi3eac2Ak) - Kunal Kushwaha
+* [Intro to DevOps](https://www.udacity.com/course/intro-to-devops--ud611) - Nutani (Udacity)
```
Comment (davorpa) [inline courses/free-courses-en.md:375]: Add instructors

```suggestion
* [Intro to DevOps](https://www.udacity.com/course/intro-to-devops--ud611) - Nutani, `tch.:` Karl Krueger, `tch.:` Dwayne Lessner, `tch.:` Gundega Dekena (Udacity)
```

Comment (DrishtiJ) [reply courses/free-courses-en.md]: Done

File: courses/free-courses-en.md [hunk 3]
```diff
@@ -372,6 +372,7 @@
 ### DevOps
 
 * [DevOps Bootcamp](https://www.youtube.com/playlist?list=PL9gnSGHSqcnoqBXdMwUTRod4Gi3eac2Ak) - Kunal Kushwaha
+* [Intro to DevOps](https://www.udacity.com/course/intro-to-devops--ud611) - Nutani, `tch.:` Karl Krueger, `tch.:` Dwayne Lessner, `tch.:` Gundega Dekena (Udacity)
```
Comment (eshellman) [inline courses/free-courses-en.md:375]:
```suggestion
* [Intro to DevOps](https://www.udacity.com/course/intro-to-devops--ud611) - Karl Krueger, Dwayne Lessner, Gundega Dekena (Udacity)
```
no, we don't need `tch.` annotations. Sorry for the confusion.

Comment (DrishtiJ) [reply courses/free-courses-en.md]: @eshellman ohk, have committed the change.

**Review by davorpa — APPROVED** [review]

**Pull request merged**